types/types.proto

```diff
@@ -161,7 +162,8 @@ message TxCreateDeployment {
   bytes  tenant  = 1 [(gogoproto.customtype)="github.com/ovrclk/akash/types/base.Bytes",(gogoproto.nullable) = false];
   uint64 nonce = 2;
   int64 orderTTL = 3;
-  repeated GroupSpec groups = 4;
+  bytes version = 4;
```

Comment:
`[(gogoproto.customtype)="github.com/ovrclk/akash/types/base.Bytes",(gogoproto.nullable) = false]`?

Reply:
I thought base.Bytes is only relevant for addressing?